smartmodule/cargo_template/cargo-generate.toml

```diff
@@ -1,15 +1,15 @@
-[placeholders.smartmodule-version]
+[placeholders.fluvio-smartmodule-cargo-dependency]
 type = "string"
-default = "git = \"https://github.com/infinyon/fluvio.git\""
-prompt = "SmartModule Version"
+default = "\"0.2.5\""
```

Comment:
This should be 0.3.0 or greater

Reply:
I was considering removing this default so we don't have to maintain it. `smdk generate` will automatically select the newest stable from crates.io.

Reply:
That make sense.  We can publish SmartModule crates since they seems stable